[Docs] Update dev client docs (#13447)
expo/expo: docs/pages/clients/getting-started.md

<Tab >
<TerminalBlock cmd={["expo init # if you don't already have a Managed Workflow project", "yarn add expo-dev-client"]}  />

> > You can also improve error messages what will be helpful during the development process. To do it, you need to add `import 'expo-dev-client';` on top of your `App.{js|tsx}` file. [Learn more](installation.md#add-better-error-handlers).

Reviewer: ```suggestion
> You can also improve error messages so that they will be more helpful during development. To do so, add `import 'expo-dev-client';` to the top of your `App.{js|tsx}` file. [Learn more](installation.md#add-better-error-handlers).
```
does this "learn more" link work?

Author: It does! at least in development

If you have an existing project, you'll need to [install the package and make a few changes](installation.md) to your `AppDelegate.m`, `MainActivity.java` and `MainApplication.java`.

> Custom clients use deep links to open projects from the QR code. If you had added a custom deep link schema to your project, your client will use it. However, if this isn't the case, you need to configure the deep link support for your application. The `uri-scheme` package will do this for you once you have chosen a scheme.

Reviewer: ```suggestion
Custom clients use deep links to open projects from the QR code. If you had added a custom deep link schema to your project, your client will use it. However, if this isn't the case, you need to configure deep link support for your application. The `uri-scheme` package will do this for you once you have chosen a scheme.
```
"schema" or "scheme"?
